zipkin-collector/kafka/src/main/java/zipkin2/collector/kafka/KafkaCollectorWorker.java

```diff
@@ -83,7 +83,7 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
             assignedPartitions.set(Collections.unmodifiableList(new ArrayList<>(partitions)));
           }
         });
-      LOG.info("Kafka consumer starting polling loop.");
+      LOG.info("Kafka consumer starting polling loop for topic {}", topics);
```

Comment:
since we're using a log line anyway, should we say the boostrap servers (either from init or something the driver says)? also, do we support multiple topics now? if so "for topics"

Reply:
On second thought I have reverted this, raising the log level gives all information necessary to troubleshoot the configuration.